### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Skobraf**
Finishing the test cases

### CODE REVIEW

**Comment by jklemon17** [comment]
Please fill out the README file with a description of the project, the rules of the game, and how to run the game locally.

**Comment by jklemon17** [comment]
I know this was not in the Odin Project instructions (I remember being frustrated by it), but the project should be divided in separate files and directories: the code to actually run the game (your last 5 lines) should be in a 'bin' folder with the file named something like tictactoe.rb; the rest of the code should be in a 'lib' folder, with each class having a separate file. 

Doing all this will allow you to make the bin file executable, to run on the command line similar to other programs. Here's a link on how to make it executable once the file structure is correct: https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/

Let me know if any of this needs more clarification.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Tic-Tac-Toe-ruby
 ## Description
+also known as noughts and crosses or Xs and Os) is a paper-and-pencil game for two players, X and O, who take turns marking the spaces in a 3×3 grid. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row wins the game.
```
Comment (jsgilberto) [inline README.md]: We need to enhance the text below Description, also the title.
Capitalize and remove outmatching parenthesis!

**Comment by jklemon17** [comment]
When you split out the files, try to remove all user interaction ('puts' and 'gets') from your game classes by creating a separate Messages class (and file) containing the methods for getting and displaying text. The reason for this is greater flexibility. (Perhaps in the future, the game could be a web app instead of command line, in which case messages would need to be re-written for the new interface, but the rest of the code could remain untouched.)

**Review by jsgilberto — CHANGES_REQUESTED** [review]
Capitalize and remove outmatching parenthesis

**Review by jsgilberto — APPROVED** [review]

**Comment by jklemon17** [comment]
The 'loop_game' method does way too much. The goal in programming is for each method to do exactly one thing. While this may not always be completely possible or practical, loop_game is doing too much. I recommend moving most of its logic outside the TicTacToeGame class, to the bottom of the file (actually, to the new 'bin' file which you will create). You may need to write some new methods for the TicTacToeGame class to make this possible, but it will improve the program overall. Remember, try to make each method do *one* thing.

**Comment by jklemon17** [comment]
The 'validate_input' method should be moved to the TicTacToeGame class.

**Comment by jklemon17** [comment]
Methods should consistently use snake_case, instead of camelCase. You currently have a mix of the two.

**Comment by jsgilberto** [comment]
thanks @jklemon17 we will make the changes

**Pull request merged**